src/spendings/spendings-card/spending-card.mapper.ts

```diff
@@ -0,0 +1,15 @@
+import { ISpendings } from '@/api/types/ISpendings'
+
+import { SpendingsCardProps } from './spendings-card'
+
+export function mapSpendingsToProps(items: ISpendings[]): SpendingsCardProps[] {
```

Comment:
добавь проверку на undefined, если items undefined возвращай пустой массив
```suggestion
export function mapSpendings(items?: ISpendings[]): SpendingsCardProps[] {
```